WordPressAuthenticator/Extensions/WPStyleGuide+Login.swift

```diff
@@ -177,7 +178,7 @@ extension WPStyleGuide {
                                         width: imageSize.width, height: imageSize.height)
 
         let buttonString = NSMutableAttributedString(attachment: appleAttachment)
-        let appleTitle = NSLocalizedString(" Continue with Apple", comment: "Button title. Tapping begins log in using Apple. There is a leading space to separate it from the Apple logo.")
+        let appleTitle = "\u{00a0}" + NSLocalizedString("Continue with Apple", comment: "Button title. Tapping begins log in using Apple. There is a leading space to separate it from the Apple logo.")
```

Comment:
We can probably remove `There is a leading space to separate it from the Apple logo.` from the string comment and make it a code comment like you did Google.